packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/FishjamClientInternal.kt

```diff
@@ -162,6 +161,7 @@ internal class FishjamClientInternal(
                 PeerNotifications.PeerMessage.AuthRequest
                   .newBuilder()
                   .setToken(connectConfig.token)
+                  .setSdkVersion("mobile-0.6.0")
```

Comment:
Can we read this value from config files?

Reply:
🫡 https://linear.app/swmansion/issue/FCE-952

Reply:
Thanks!
I also updated release instruction to keep it in mind in upcoming release
https://github.com/fishjam-cloud/internal-docs/wiki/Mobile-SDK-release